### ISSUE TITLE
[Documentation] Optimizations and dependence on hardware

### ISSUE THREAD

**Issue by azsh1725** [microsoft/onnxruntime#15529]
### Describe the documentation issue

Am I correct in assuming that since extended optimizations are applied to nodes with a certain EP, then they are dependent on hardware? 

So, if I'm understanding correctly, does that mean extended optimizations are best applied at runtime, i.e. in this case only in online mode, and offline mode is not available, because there may be performance problems?

Thanks in advance.

### Page / URL

https://onnxruntime.ai/docs/performance/model-optimizations/graph-optimizations.html#extended-graph-optimizations

**Comment by natke** [comment]
Yes the extended optimizations are applied at runtime @azsh1725.

Let us know if that answers your question and tell us a bit more about your use case.
